tests/read_csv_test.py

```diff
@@ -40,3 +41,13 @@ def test_read_csv_v1(
     result = nw_v1.read_csv(filepath, native_namespace=native_namespace)
     assert_equal_data(result, data)
     assert isinstance(result, nw_v1.DataFrame)
+
+
+def test_read_csv_kwargs(
```

Comment:
this is failing on the minimum versions

can we use `skipif` with the pandas version < (1,5)? there's some other tests that do something like this